src/css/custom.scss

```diff
@@ -709,3 +709,7 @@ a {
   transform: rotate(180deg) !important;
 }
 /* END DETAILS MARKERS */
+.avatar__photo {
```

Comment:
Consider using a custom class to set these styles so the changes are only applied to the `author__avatar`